apps/evm-bridge/src/main.tsx

```diff
@@ -83,11 +83,29 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
                                 >
                                     <ThemeProvider appId="IOTA-evm-bridge">
                                         <RainbowKit>
-                                            <CookieManagerProvider>
-                                                <App />
-                                                <Toaster />
-                                                <CookieDisclaimer />
-                                            </CookieManagerProvider>
+                                            <App />
+                                            <Toaster />
+                                            <Disclaimer
+                                                onClose={() => {
+                                                    handleConsentAccepted();
+                                                }}
+                                            >
+                                                <div className="text-body-md text-neutral-10 dark:text-neutral-92">
```

Comment:
```suggestion
                                                <div className="text-body-md text-iota-neutral-10 dark:text-iota-neutral-92">
```
The same as in the Explorer